core-bundle/tests/Command/MigrateCommandTest.php

```diff
@@ -236,12 +255,15 @@ public function testDoesNotExecuteWithDryRun(string $format): void
         if ('ndjson' === $format) {
             $this->assertSame(
                 [
-                    ['type' => 'migration-pending', 'name' => 'Migration 1'],
-                    ['type' => 'migration-pending', 'name' => 'Migration 2'],
-                    ['type' => 'migration-pending', 'name' => 'Runonce file: runonceFile.php'],
+                    [
+                        'type' => 'migration-pending',
+                        'names' => ['Migration 1', 'Migration 2', 'Runonce file: runonceFile.php'],
+                        'hash' => 'fd96e0795abea843b443ccd39c746b5f1491c45131611f14a7c5bfb518824252',
+                    ],
```

Comment:
it seems we are (majorly) changing the `ndjson` output to add the hash, is that really ok in a patch release?

Reply:
IMO yes. Nobody used `ndjson` and the format itself was added in a patch release too.